languages/javascript/hron.js

```diff
@@ -82,10 +94,10 @@
 		var result;
 		if (match) {
 			state.skipLine("Value_Begin", match[1]);
-			result = { key: match[1] }
-			++state.currentIndent;
+			result = { key: match[1] };
+			state.currentIndent++;
 			result.value = deserializeValueLines(state);
-			--state.currentIndent;
+			state.currentIndent--;
```

Comment:
Interestingly, the static code analysis complained about this.

Date: Wed, 18 Mar 2015 13:46:38 -0700
From: notifications@github.com
To: hron@noreply.github.com
CC: danielbrannstrom@hotmail.com
Subject: Re: [hron] Improved bundling of javascript lib (#32)

In languages/javascript/hron.js:

>  			result.value = deserializeValueLines(state);
> -			--state.currentIndent;
> +			state.currentIndent--;

minify doesn't allow predecrement?


—
Reply to this email directly or view it on GitHub.